static/menu.js

```diff
@@ -53,6 +135,7 @@ const updateMenuPositionForSubMenu = (currentMenuSupplier) => {
         currentMenu = newMenu;
         newMenu.className += " pure-menu-active";
         backdrop.style.display = "block";
+        loadReleases();
```

Comment:
This looks like it will load the releases when _any_ menu is opened, even on non-crate pages.

Reply:
I modified `loadReleases` to look if there is the element where to put the release list so that it won't make the AJAX query if it's not necessary.